src/passes/Print.cpp

```diff
@@ -88,7 +88,8 @@ struct PrintExpressionContents
   void visitLoop(Loop* curr) {
     printMedium(o, "loop");
     if (curr->name.is()) {
-      o << ' ' << curr->name;
+      o << ' ';
+      printName(curr->name, o);
```

Comment:
How would you feel about making `printName` a wrapper struct for `Name` that has an overload for `<<` with ostream rather than keeping it a function? That would allow us to do something like `o << ' ' << PrintName(curr->name);` rather than having to split that print across two lines. The same trick could be done with `PrintLocal`.

Reply:
I think we could that for both printLocal and printName as a followup maybe?  But we already have things like `printMedium` which don't follow this pattern.